### DISCUSSION TITLE
Declarative window rules for moving to defined virtual desktop

### DISCUSSION THREAD

**Discussion by hasrthur** [nix-community/plasma-manager#343] [Q&A] [answered]
plasma-manager has possibility to declaratively set desktop names and window rules but to move the window to a specific desktop ones needs a desktop id. At this point I hardcode those ids. Is there any better solution for this than hardcoded values? 

P.S. If I don't hardcode the ids in config they are changed with every login since I presume I use override option

**Comment by HeitorAugustoLN** [comment]
Maybe we could implement generating those desktop ids via `uuidgen` command, but I need to look into it yet. But for now there is not any other solution

**Comment by HeitorAugustoLN** [comment, marked as answer]
This was fixed in #408. Now you can reference any desktop with the following pattern: `Desktop_${desktop_number}`
